Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -951,7 +951,22 @@ void vtkMRMLSliceNode::ReadXMLAttributes(const char** atts)
     // Layout color is set in Superclass
     layoutColorFound = true;
   }
+
+  // Read Dimensions only if the current Dimensions are the default ones
+  if (this->Dimensions[0] == 256 && this->Dimensions[1] == 256 && this->Dimensions[2] == 1)
```

Comment:
Why do we read/copy only when default value? Do we really ever want to read this from file?

(256,256,1) is a valid value, so if we want to make decision based on it then we would need to change the default to some invalid value (0,0,0) or maybe (-1,-1,-1).

Reply:
Done in https://github.com/Slicer/Slicer/pull/8370/commits/52914830db2a06573ec1446dca932bb51ba8e9c9